src/machine.adoc

```diff
@@ -2336,6 +2336,12 @@ For RV64, naturally aligned 64-bit memory accesses to the `mtime` and
 ....
 
 
+The `time` CSR is a read-only shadow of the memory-mapped `mtime` register.
+When XLEN=32, the `timeh` CSR is a read-only shadow of the upper 32 bits of the
+memory-mapped `mtime` register, while `time` shadows only the lower 32 bits of
+`mtime`.
```

Comment:
Perhaps I'm misreading this diff (I'm using the GitHub web interface). But this text appears to be redundant with text in lines 1578-1582.

Reply:
It is redundant.  At least last time we discussed it, the ARC thought that this was actually the right place to include this text, but that deleting it from the other location might result in some confusion, and so duplicating it was the way to go.